readme.md

```diff
@@ -31,10 +31,12 @@ button.addEventListener('click', () => {
 
 ## API
 
-### copy(text)
+### copy(text [, target])
 
 Copy `text` to the clipboard.
 
+Optionally specify a `target` DOM element (`document.body` by default) where the temporary, behind-the-scenes `textarea` should be appended. Use in cases where you need to stay within a focus trap for instance.
```

Comment:
Can you mention the "modal" case as an example of "focus trap"? Just to make it clear where this is most commonly useful.